test/xapi/track_test.rb

```diff
@@ -24,27 +24,11 @@ def test_default_track
     assert_equal slugs, track.slugs
 
     # default test pattern
-    files = [
-      "file.ext",
-      "test.ext",
-      "file_test.ext",
-      "TestFile.ext",
-      "spec.ext",
-      "sub/test/file.ext",
-      "sub/TestDir/file.ext",
-    ]
-    assert_equal %w(file.ext spec.ext), files.reject {|f|
-      f =~ track.test_pattern
-    }
+    assert_equal(/test/i, track.test_pattern)
```

Comment:
> the 'these filenames get filtered correctly' test should be there.

Yeah, that's what I'm looking for.

Sorry, I don't recall all of the details of the code (I'm forever reading for context). In terms of testing that the default is set, I think this is fine. And with that in mind, the other test also makes sense.